apps/numpy_dlpack/dlpack/dlpack.py

```diff
@@ -0,0 +1,102 @@
+import ctypes
+
+_c_str_dltensor = b"dltensor"
+
+
+class DLDeviceType(ctypes.c_int):
```

Comment:
Shall we also add docstrings to these classes correspondingly?

Reply:
Added. Please provide feedback.